data/transactions/logic/eval.go

```diff
@@ -1325,6 +1326,14 @@ func (cx *evalContext) txnFieldToStack(txn *transactions.Transaction, field TxnF
 		sv.Uint = uint64(txn.ApplicationID)
 	case OnCompletion:
 		sv.Uint = uint64(txn.OnCompletion)
+	case Args:
```

Comment:
We don't need `txn Args N` because we already have the `arg N` ops

Reply:
+1 to Brian, `arg N` is already in place. `NumArgs` looks legit.

Reply:
Hi @algorandskiy You are right that `arg N` is the same `txna Args N`, but there is no way to do `gtxna Args N` without this patch. Meaning that txn 0 cannot look at txn 1's arguments.

If you'd like to see an example of this checkout --- https://github.com/reach-sh/reach-lang/blob/master/examples/overview/build/index.main.mjs#L166 --- and if you'd like an explanation, you can watch my video --- https://www.youtube.com/watch?v=6QXISWGoul4 --- where I explain how I compile Reach to Algorand

Reply:
Yeah, now I see why, thanks for the explanation. I'm a bit worried about the design and consistency - currently `txna`/`gtxna` access transaction fields not signature fields. Sounds like we either want `txn` accessing all signature fields or to have another opcode for signature fields accessing. Anyway, thank you for the proposal! Now it needs to want to be batched with other for TEAL v3.

Reply:
I don't have a strong opinion on the name. I can see a case for (A) `txn/gtxn/txna/gtxna` are for _all transaction_ information and (B) making sure that users know that a transaction's signature information is in a sub-category from the transaction's general information. I, perhaps false, feel pretty confident in my understanding of Algorand and I don't find it intuitive to think about them differently, so that's why I prefer (A)